upgrade/scripts/upgrade/ncn-upgrade-worker-storage-nodes.sh

```diff
@@ -263,7 +263,7 @@ function printCmdArgs() {
 function getUnsucceededRebuildWorkflows() {
   res_file=$(mktemp)
   local labelSelector="node-type=${nodeType},workflows.argoproj.io/phase!=Succeeded"
-  http_code=$(curl -s -o "${res_file}" -w "%{http_code}" -k -XGET -H "Authorization: Bearer $(getToken)" "${baseUrl}/apis/nls/v1/workflows?labelSelector=${labelSelector}")
+  http_code=$(curl -s -m 10 -o "${res_file}" -w "%{http_code}" -k -XGET -H "Authorization: Bearer $(getToken)" "${baseUrl}/apis/nls/v1/workflows?labelSelector=${labelSelector}")
```

Comment:
Without `-m|--max-time`, under certain conditions curl command may hang forever instead of simply returning error.